tailwind.config.js

```diff
@@ -1,11 +1,21 @@
+const colors = require("tailwindcss/colors")
+
 module.exports = {
-  purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
```

Comment:
This change is not related to this PR, but it fixes the formatting style to double quotes.